source/blog/2017-11-07-fedora-atomic-27-on-multiarch.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Fedora 27 Atomic Host Availability on Multiple Architectures
+author: sinnykumari
+date: 2017-11-07
+published: true
+comments: false
+tags: atomic, fedora, multi-arch, ppc64le, aarch64
+published: true
+---
+
+We are proud to announce that multiple architectures are now supported with Fedora 27 Atomic Host release! Now, along with x86_64 architecture Atomic Host is also available on 64 bit ARM (AArch64) and PowerPC Little Endian (ppc64le). Both aarch64 and ppc64le architectures will receive Atomic OSTree updates in the same way x86_64 does.
+Atomic Host provides immutable operating system tree (OStree) with rollback facility. To learn more, see [Project Atomic website](http://www.projectatomic.io/).
+
+# Download
+Fedora Atomic Host for aarch64 and ppc64le are available in both ISOs and cloudImages-
+- aarch64
+    - [ISO](link)
+    - [QCOW](link)
+    - [Raw image](link)
+
+- ppc64le
+    - [ISO](link)
+    - [QCOW](link)
+    - [Raw image](link)
+
+# Installation
+Atomic Host can be installed as per user preference where they want to install it - on bare metal, virtual machine, cloud setup and so on.
+If you have some aarch64 or ppc64le hardware, we'd love for you to take Atomic Host for a spin. One quick way is to run `virt-install` on one of theses platforms:
```

Comment:
What's virt-install?  What are the requirements for it?  What software provides support for it?

Reply:
Okay, will add details about virt-install.